source/sourcing_and_processing_data/processors/expand_json_array.rst

```diff
@@ -0,0 +1,75 @@
+Expand JSON array processor
+===========================
+
+This processor transposes rows containing a JSON array into several rows with a new column containing each value of the array.
+
+Setting the processor
+---------------------
+
+The parameter "iJSON rule to array" works exactly like in the "Extract from JSON" processor and should contain the array to transpose (represented with the iJSON rule ``.item``).
+
+- If the field contains the JSON array directly, just put ``item`` as an iJSON rule.
+- If the final element is an array, the iJSON rule must end with ``.item``, meaning that the reached object should be treated as an array of items in the iJSON syntax.
+- If you want to keep going into the items inside the array, you can keep adding key names after the ``.item``, but be careful to check that this path is valid for every object in the array.
+
+To set the parameters of the Expand JSON array processor, follow the indications from the table below.
+
+.. list-table::
+  :header-rows: 1
+
+  * * Label
+    * Description
+    * Example
+  * * JSON array field
+    * Name of the field that holds the JSON array
+    * data
+  * * iJSON rule to array
+    * iJSON rule to iterate in the JSON array above. An iJSON rule is built with the names of all the field from the JSON root to the data to extract, separated with a dot.
+    * block.metaB
+  * * Output field
+    * Name of the field that will contain the extracted element
+    *
+
+Example
```

Comment:
Not sure about the "formatting" of the section. We may want to add something visual to explain the result. We can talk about it ;)